coremis_app_integration/authorization/custom_basic.py

```diff
@@ -0,0 +1,111 @@
+"""
+Provides various authentication policies.
+"""
+import base64
+import binascii
+
+from django.contrib.auth import get_user_model
+from django.utils.translation import gettext_lazy as _
+from rest_framework.authentication import (
+    authenticate,
+    BasicAuthentication,
+    get_authorization_header
+)
+
+from rest_framework import exceptions
+
+
+FAILED = {
+  "status": False,
+  "count": 1,
+  "result": "",
+  "message": "ERROR",
+}
+
+
+DETAILS = {
```

Comment:
You should consider changing dicts data formats to the Dataclasses. They'll give more control over structure.

Reply:
changed